VCFv4.3.tex

```diff
@@ -1509,8 +1509,8 @@ \subsubsection{Site encoding}
                         Must be the length of the REF allele or the declared length of a symbolic allele respecting
                         the END attribute \\ \hline
 QUAL      & float     & Variant quality; 0x7F800001 for a missing value \\ \hline
-n\_allele\_info & int32\_t & The low 16 n\_allele bits represent the number of REF+ALT alleles in this record and the
-                             high 16 n\_info bits represent the number of VCF INFO fields in this record \\ \hline
+n\_allele & uint16\_t & The number of REF+ALT alleles in this record \\ \hline
+n\_info   & uint16\_t & The number of INFO fields in this record \\ \hline
```

Comment:
The HTSlib implementation has `n_info` first then `n_allele`, both before and after samtools/htslib#1023's rewrite. The text about low and high bits is incorrect and was only added in the second commit on this PR (3a931e1c2a8fa53764f0d008fcf9907d7010b184) — **master** just has “`n_info`, where `n_allele` is the number of REF+ALT alleles in this record, and `n_info` is the number of VCF INFO fields present in this record”.

Reply:
Oops.  Should be fixed now.